modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/DistributionZoneWithKeywordSyntaxSqlDdlParserTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.sql.engine.sql;
+
+import static org.apache.ignite.internal.sql.engine.sql.DistributionZoneSqlDdlParserTest.assertThatZoneOptionPresent;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.ignite.internal.sql.engine.prepare.ddl.ZoneOptionEnum;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests to verify parsing of the DDL "ZONE" commands contains WITH syntax.
+ */
+public class DistributionZoneWithKeywordSyntaxSqlDdlParserTest extends AbstractParserTest {
```

Comment:
`DistributionZoneWithKeywordSyntaxSqlDdlParserTest` -> `DistributionZoneObsoleteSyntaxSqlDdlParserTest `

we must make it clear that this syntax is obsolete and no one should use it